homeassistant/components/camera/media_source.py

```diff
@@ -94,7 +97,9 @@ async def async_browse_media(
                     identifier=camera.entity_id,
                     media_class=MediaClass.VIDEO,
                     media_content_type=content_type,
-                    title=camera.name,
+                    title=cast(
```

Comment:
The previous implementation left an empty title for cameras that took their name from the device. This seems to be a better choice, but if there's a better way to get the name I'll change it.